### PR TITLE
Remove space before colon and question mark in strings

### PR DESCRIPTION
**Pull request by amire80**
Space is necessary before ':' and '?' in the French language,
but it's not necessary in English and in most other languages.

### CODE REVIEW

File: resources/i18n/en.json [hunk 1]
```diff
@@ -117,7 +117,7 @@
     "no":"no",
     "no-filter":"no filter",
     "open-link-in-web-browser":"Open link in web browser",
-    "download-dir-dialog-title":"Are you sure you want to change the download directory ?",
-    "download-dir-dialog-msg":"The new download directory path will be :\n{{DIRECTORY}}",
+    "download-dir-dialog-title":"Are you sure you want to change the download directory",
```
Comment (mgautierfr) [inline resources/i18n/en.json:120]: You have removed the question mark with the space.

**Review by mgautierfr — CHANGES_REQUESTED** [review]

**Comment by amire80** [comment]
:facepalm: 

I put the question mark back. Thanks for the careful review! :)

**Comment by kelson42** [comment]
Actually, and this is something we agreed with @juuz0, to me we should remove the colon character on the settings panel.

**Comment by kelson42** [comment]
Thanks @amire80 for your PR, I have taken the essence of it in #725.

**Comment by amire80** [comment]
Thanks!

**Pull request closed without merging**